apps/cli/lib/parser/mana_parser.ex

```diff
@@ -0,0 +1,151 @@
+defmodule CLI.Parser.ManaParser do
+  @moduledoc """
+  Parses for command line arguments for the CLI.
+  """
+
+  @default_chain_name "ropsten"
+  @default_no_discovery false
+  @default_no_sync false
+  @default_bootnodes "from_chain"
+  @default_debug false
+
+  @doc """
+  Parsers args for syncing
+
+  ## Options:
+    * `--chain` - Chain to load data from (default: ropsten)
+    * `--no-discovery` - Perform discovery (default: false)
```

Comment:
I think this isn't valid anymore

Reply:
hm, I think those are accurate, the other comments below in `tasks/mana.ex` were incorrect. I've updated the docs there.